Lesson1.ipynb

```diff
@@ -170,29 +159,22 @@
    "source": [
     "## Now, it's your turn to add a new cell!\n",
     "\n",
-    "Click on \"+\" button to add a new cell. Try to add:\n",
-    "* one new Markdown cell\n",
-    "* one new Code cell"
+    "* Click on \"+\" button above to add a new cell\n",
+    "* Add a new Code cell and create your own simple print command or math operations in Python"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Got that? Let's talk about different File Types\n",
+    "## Method 2: Run Python script from the Python terminal\n",
     "\n",
-    "* A file with only Markdown text has the file extension .md (e.g. README.md)\n",
-    "* A file with only Python code has the file extension .py\n",
-    "* A file (like this one!) which has a mixture of Markdown text and Python code is an iPython notebook, and has the file extension .ipynb"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Now let's save your file\n",
+    "1. Use your normal text editor and create a new file named *HelloWorld.py*. Within the file, there should be the following line: print (\"Hello World\")\n",
```

Comment:
@puikwan I think this part could be like: 
- Click the "+" button on the navigation pane (left column), a new **Launcher** tab will appear
- Select **TextEditor**, a new file **untitled.txt** is created
- From the navigation pane, right click and rename the file to **hello.py** (replace .txt too), notice that the file icon change to that of Python's
- Enter `print("Hello, world!")` in hello.py, and save by clicking save button or press Ctrl+S (Windows) or Cmd+S (OSX)